plugins/aea-ledger-cosmos/setup.py

```diff
@@ -24,7 +24,7 @@
 
 
 setup(
-    name="aea_ledger_cosmos",
+    name="aea-ledger-cosmos",
```

Comment:
this is more correct. The previous one is the name of the Python package, whereas `name` here should be the name of the PyPI package.